components/web/goaccess/goaccess.p5m

```diff
@@ -11,6 +11,7 @@
 
 #
 # Copyright 2016 Adam Stevko
+# Copyright 2016 Pablo Sarria
 #
```

Comment:
2020 :)

Reply:
Thanks AndWac, it's fixed now, i absolutely forget the year !!! :D